velox/exec/HashAggregation.cpp

```diff
@@ -180,12 +180,12 @@ void HashAggregation::updateRuntimeStats() {
     }
   }
 
-  runtimeStats["hashtable.capacity"] = RuntimeMetric(hashTableStats.capacity);
-  runtimeStats["hashtable.numRehashes"] =
+  runtimeStats[std::string(kCapacity)] = RuntimeMetric(hashTableStats.capacity);
```

Comment:
it is a bit tedious to wrap names in std::string? Can we figure out how to not require that?

```
runtimeStats[kCapacity] = RuntimeMetric(hashTableStats.capacity);
```

Reply:
I will put them into the HashTable class as static members and use std::string.